### PR TITLE
Small code clean up

### PR DESCRIPTION
**Pull request by erictuvesson**
Hey,

The only change I have done here is add this to the **Server**.
```cs
void Broadcast(byte[] packet)
```

### CODE REVIEW

**Comment by markiodev** [comment]
Thanks for your contribution!

If you send a byte[] to the clients and there is no data about which type of packet you are sending then the client will not be able to handle it. I should add some more information to the wiki about packet structure but it looks something like:

PacketTypeName_Size
PacketData_Size
PacketTypeName
PacketData

If you are relaying the above packet it would work, otherwise it will be simply unhandled on the client end.

What are you trying to achieve with broadcasting a byte[]?

File: Networker/Client/Client.cs [hunk 1]
```diff
@@ -42,78 +42,77 @@ public class Client : IClient
...
+                        }
+                    }
+
+                    this.udpClient = null;
+                });
             }
         }
 
-        public long Ping()
+        public long Ping(int timeout)
```
Comment (erictuvesson) [inline Networker/Client/Client.cs:84]: I added the parameter here, there is a default in the interface.

File: Networker/Client/Abstractions/IClient.cs [hunk 2]
```diff
@@ -7,11 +7,16 @@ public interface IClient
     {
         EventHandler<Socket> Connected { get; set; }
         EventHandler<Socket> Disconnected { get; set; }
-        void Connect();
+
         void Send<T>(T packet);
+        void Send(byte[] packet);
```
Comment (erictuvesson) [inline Networker/Client/Abstractions/IClient.cs]: I added this method too.

**Comment by erictuvesson** [comment]
I want to be able to skip the serializer and send just bytes.

_Some packets I want to send are serialized in a more optimal._

I never ended up trying it out since I also want to do the same with the TCP socket and make it easier to send packets there.

**Comment by erictuvesson** [comment]
_Note: While working on this I noticed that the ```ProtobufNet``` is not working_

**Comment by markiodev** [comment]
I have an idea for sending the byte[] (and other data clr types) which I want to give a bit more thought to. The other changes all look good.

I'll look at why ProtobufNet is not working as I continue writing my tutorial.

**Review by markiodev — APPROVED** [review]

**Pull request merged**